schema/rpc.fbs

```diff
@@ -71,10 +72,18 @@ enum ResetType: uint8 {
 }
 
 
+enum ResetStatus: uint8 {
+    TRIGGERED
+}
+
 table ResetRequest {
     reset_type: ResetType;
 }
-table ResetResponse {}
+table ResetResponse {
+    reset_type: ResetType;
+    status: ResetStatus;
```

Comment:
A reset request sends a `ResetType`. We respond with a reset type and a status.

Can you talk me through some examples of how that works?

Reply:
For example, when TabDetection triggers a full Reset, the server will send a ResetResponse( with status = triggered, and reset_type = Full reset), this will applied to shortcut key trigger and GUI button.

Later, we can also add status about starting reset and completed reset

Reply:
OK, I think I understand what you intended to go with now.

Lets make this change:
```
enum ResetStatus: uint8 {
    Started,
    Finished,
}
```